pkg/services/dashboardsnapshots/models.go

```diff
@@ -29,11 +29,11 @@ type DashboardSnapshot struct {
 
 // DashboardSnapshotDTO without dashboard map
 type DashboardSnapshotDTO struct {
-	ID          int64  `json:"id" xorm:"id"`
+	ID          int64  `json:"-" xorm:"id"`
```

Comment:
The entire purpose of this `DashboardSnapshotDTO` seems to be returning the search response, we could remove those fields instead of changing the JSON name field

Reply:
Yes -- but i am reluctant to make too big of change without a concrete destination :).   With this minimal change, we keep all internal logic the same but do not leak the the database ids over the public api